src/test/scala/edu/berkeley/cs/rise/opaque/OpaqueOperatorTests.scala

```diff
@@ -283,6 +283,17 @@ trait OpaqueOperatorTests extends FunSuite with BeforeAndAfterAll { self =>
     df.sort($"x", $"y").collect
   }
 
+  testAgainstSpark("sort with null values") { securityLevel =>
+    val data : Seq[Tuple1[Integer]] = Random.shuffle((0 until 256).map(x => {
```

Comment:
Style: No space before the `:` ([ref](https://docs.scala-lang.org/style/types.html#annotations))